src/lib/utils.ts

```diff
@@ -18,6 +18,24 @@ interface IRPCError {
     data?: string;
 }
 
+export interface DataPack {
+    encode(value: any): any;
+    decode(value: any): any;
```

Comment:
Should we define return types here instead of using `any`?

Reply:
> Should we define return types here instead of using `any`?

Thanks, based on your suggestion I improved the code and updated the related API documentation. Please review it again